### ISSUE TITLE
Add Facet for Event Class Data Type

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-marklogic#139]
**Problem Description:**
A facet to filter for Event data types (activity or period) in Simple Search or Advanced Search does not currently exist.

**Expected Behavior/Solution:**
Create facet/results for the Event scope and all its values in both Simple and Advanced Search. 
These results will pull from the "Type" listed in each JSON event record.

Example of record with Period Type:

```
{
"id": "https://lux-front-tst.collections.yale.edu/data/activity/85d48e7c-32af-4809-9c69-fa8e49fe51bc",
"type": "Period",
"_label": "War and intervention, 1950-",
```

Help text will also be added for each value. 

**Requirements:**
The following values will be pulled from the Type value in the JSON record:
- Value: Period | 'Period'
- Value: Activity | 'Activity'
- the following Help Text will be added for each value selection: 
   - **Event Class:**  Search for records categorized as either a "Period" or an "Activity"
   - **Period:** Search for Events that have the class of Period, which consists of a Time Period, as opposed to other human-caused activities.
   - **Activity**: Search for Events that have the class of Activity, which consists of events such as Exhibitions or other collections-related activities. 

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] ~Wireframe/Mockup - Mike~
- [x] Committee discussions - Sarah
- [x] Feasibility/Team discussion - Sarah
- [x] Backend requirements - TBD
- [ ] ~Frontend requirements- TBD~
- [ ] ~Are new regression tests required for QA - Amy~
- [x] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**

- Please review UAT ex from FE ticket | https://github.com/project-lux/lux-frontend/issues/175

**Dependencies/Blocks:**
Blocking/dependency: https://github.com/project-lux/lux-frontend/issues/175

**Related Github Issues:**

- Issues that contain similar work but are not blocking or being blocked by the current issue.

**Related links:**

- These links can consist of resources, bugherds, etc.

~**Wireframe/Mockup:**~
~Place wireframe/mockup for the proposed solution at end of ticket.~

**Comment by clarkepeterf** [comment]
@prowns @jffcamp @roamye 

I have the following for the advanced search label and the help text:

      label: 'Event Class',
      helpText:
        'Search only records that are either a "Period" or an "Activity"',

Is this correct?

**Comment by roamye** [comment]
@clarkepeterf 

the tix has been updated within the issue to include the help text for Event Class. 
Help txt: Search for records categorized as either a "Period" or an "Activity"

**Comment by roamye** [comment]
Approved by UAT - however this tix will be closed on the 07/08 cycle when the FE can be tested.

**Comment by roamye** [comment]
Not closing - but I have included this in the release notes for 7/1.

**Comment by roamye** [comment]
In prod, the selection of a event class: period or any value does not update for the help text provided above.

after discussion with @clarkepeterf This is because this functionality is not supported at the moment. A new ticket will be created for the new functionality to be supported.

Besides this everything else looks good in PROD. Closing.
